src/utils/ERACommonsUtils.js

```diff
@@ -23,6 +10,7 @@ export const nihAccountInstructions = () => {
  * care about and parse the era commons part as a JSON object.
  * @param token
  * @returns JSON Object in the form of: {'eraCommonsUsername':String,'iat':Integer,'exp':Integer}
+ * @deprecated This function is deprecated and will be removed in future releases
```

Comment:
A bit more context seems like it'd help maintainability:

```suggestion
 * @deprecated This function is deprecated and will be removed when we fully migrate to RAS
```